articles/anomaly-detection/guides/set-anomaly-detection-preferences.md

```diff
@@ -1,34 +1,53 @@
 ---
 title: Set Anomaly Detection Preferences
-description: Learn how to set anomaly detection preferences in the Dashboard.
+description: Learn how to set anomaly detection preferences in the Dashboard for Automated Attack Protection, Brute-force Protection, and Breached Password Detection.
+toc: true
 topics:
     - security
     - anomaly-detection
     - brute-force-protection
     - breached-password-detection
+    - automatic-attack-protection
 contentType: how-to
-useCase: customize-anomaly-detection
-v2: true
+useCase: anomaly-detection
 ---
 # Set Anomaly Detection Preferences
 
-Customize the actions that occur after the triggers in the **Anomaly Detection** section on the [Dashboard](${manage_url}/#/anomaly).
+Enable, disable, and customize **Anomaly Detection** settings in the [Dashboard](${manage_url}/#/anomaly).
 
 ::: warning
 Auth0 recommends that you **do not** make changes to your anomaly detection features with the Management API.
 :::
 
 ![Anomaly Detection Dashboard](/media/articles/anomaly-detection/anomaly-detection-overview.png)
```

Comment:
Would like to update the overall dashboard with the new tile for "Automated Attack Protection"